packages/pds/src/migrate-script/db.ts

```diff
@@ -0,0 +1,96 @@
+import { Kysely, MigrationProvider, Migrator, SqliteDialect } from 'kysely'
+import SqliteDB from 'better-sqlite3'
+
+export const getDb = async (loc: string): Promise<MigrateDb> => {
+  const db = new Kysely<Schema>({
+    dialect: new SqliteDialect({
+      database: new SqliteDB(loc),
+    }),
+  })
+  const migrator = new Migrator({ db, provider: dbMigrationProvider })
+  const { error } = await migrator.migrateToLatest()
+  if (error) throw error
+  return db
+}
+
+const dbMigrationProvider: MigrationProvider = {
+  async getMigrations() {
+    return {
+      '1': {
+        async up(db: Kysely<unknown>) {
+          await db.schema
+            .createTable('status')
+            .addColumn('did', 'varchar', (col) => col.primaryKey())
+            .addColumn('pdsId', 'integer')
+            .addColumn('signingKey', 'varchar')
+            .addColumn('phase', 'integer')
```

Comment:
We might make this a not-null to match the types below.

Reply:
yup yup